presto-hive/src/main/java/com/facebook/presto/hive/HiveClientConfig.java

```diff
@@ -604,6 +631,19 @@ public HiveClientConfig setInsertOverwriteImmutablePartitionEnabled(boolean inse
         return this;
```

Comment:
Let make `isInsertOverwriteImmutablePartitionEnabled` a LegacyConfig

Reply:
To make this LegacyConfig, should property be same as @Config? Like below ? Since it is not overridden by any new property.
 @LegacyConfig("hive.insert-overwrite-immutable-partitions-enabled")
  @Config("hive.insert-overwrite-immutable-partitions-enabled")

Reply:
my bad; actually i meant having a `@Deprecated` annotation for it.